deploy/charts/README-helm3.md

```diff
@@ -0,0 +1,66 @@
+# Helm v3 support
```

Comment:
Could we move all these charts to `assets` or `dependencies` or some other dir you think make sense?